pkg/storaged/pages.jsx

```diff
@@ -608,19 +608,19 @@ export const PageTable = ({ emptyCaption, aria_label, pages, crossrefs, sorted,
                 </Card>);
         } else {
             const cols = [
-                <Td key="1" onClick={onClick}>
+                <Th scope="row" key="1" onClick={onClick}>
```

Comment:
This would be correct from the HTML side of things. I haven't looked to see if there are any stylistic implications (if it changes the look) yet, but I'll do that in a little bit.

Good catch on seeing that scope is deprecated for `<td>`! I felt it was just for `<th>` at first, but saw it used and mentioned for `<td>`, but I guess I saw old docs and skipped that part of the documentation on MDN. Thanks for finding and mentioning it!